src/Playwright/Core/ElementHandle.cs

```diff
@@ -133,7 +137,9 @@ public Task FillAsync(string value, ElementHandleFillOptions options = default)
             ["value"] = value,
             ["timeout"] = options?.Timeout,
             ["force"] = options?.Force,
+#pragma warning disable CS0612 // Type or member is obsolete
             ["noWaitAfter"] = options?.NoWaitAfter,
```

Comment:
You can drop all `noWaitAfter` on the floor, except for click, press and selectOption methods.